frontend/src/styles/base/typography.css

```diff
@@ -30,6 +30,19 @@ h4 {
   font-weight: 700;
 }
 
+h5 {
+  font-family: "Nunito", sans-serif;
+  font-size: var(--font-size-small);
+  font-weight: 700;
+}
+
+h5 {
+  font-family: "Nunito", sans-serif;
+  font-size: var(--font-size-small);
+  font-weight: 700;
+  margin: 8px;
+}
```

Comment:
Is this duplication on purpose? Otherwise could lines 33 to38 be deleted?

Reply:
This has been addressed.